reference/snapshot-restore.md

```diff
@@ -0,0 +1,107 @@
+---
+Title: Volume Restore from a Snapshot
+---
+
+Volume restore from an existing snapshot will create exact replicas of storage volumes captured at a specific point in time. They serve as an essential tool for data protection, recovery, and efficient management in Kubernetes environments. This article provides a step-by-step guide on how to create a volume restore.
+
+## Prerequisites
+
+### Step 1: Create a storage class 
+
+To begin, you'll need to create a StorageClass that defines the properties of the snapshot to be restored. Refer to [Storage Class Parameters](reference\storage-class-parameters.md) for more details. Use the following command to create the StorageClass:
+
+
+{% tabs %}
+{% tab title="Command" %}
+```text
+cat <<EOF | kubectl create -f -
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: mayastor-1-restore
+parameters:
+  ioTimeout: "30"
+  protocol: nvmf
+  repl: "1"
+  thin: "true"
+provisioner: io.openebs.csi-mayastor
+EOF
+```
+{% endtab %}
+{% tab title="YAML" %}
+```text
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: mayastor-1-restore
+parameters:
+  ioTimeout: "30"
+  protocol: nvmf
+  repl: "1"
+  thin: "true"
+provisioner: io.openebs.csi-mayastor
+```
+{% endtab %}
+{% endtabs %}
+
+> Note the name of the StorageClass, which, in this example, is **mayastor-1-restore**.
```

Comment:
Please add `Note: The thin: "true" and repl: "1" are the only supported combination.` somewhere here or above this line.